### ISSUE TITLE
APPEND x && y TO itab in old versions

### ISSUE THREAD

**Issue by FreHu** [abaplint/abaplint#635]
Abaplint does not catch this as incompatible with old versions:
```abap
DATA: itab type stringtab.
APPEND `x` && `y` TO itab.
```
https://github.com/larshp/abapGit/pull/3054
https://github.com/larshp/abapGit/pull/3056

**Comment by larshp** [comment]
740sp02, https://help.sap.com/doc/abapdocu_751_index_htm/7.51/en-us/abennews-740-operand_positions.htm

**Comment by larshp** [comment]
also related: 139
